### PR TITLE
Add Tab Completion

### LINKED ISSUE #11: Command Tab-Complete

### PR DESCRIPTION
**Pull request by qeamlgit**
Adds a simple Tab Completer, that only shows the subcommands the user has access to. Fixes #11.

### CODE REVIEW

**Comment by Trikolon** [comment]
Thanks for the PR! I'll take a closer look on the weekend.

**Review by Trikolon — APPROVED** [review]
Thanks! Just two suggestions.

File: src/main/java/com/tallcraft/chatreplay/ChatReplay.java [hunk 1]
```diff
@@ -169,6 +172,30 @@ public boolean onCommand(CommandSender sender, Command cmd, String commandLabel,
         return false;
     }
 
+    public List<String> onTabComplete(CommandSender sender, Command cmd, String label, String[] args) {
+        // the user is typing the desired subcommand
+        if(args.length == 1) {
```
[the remark this answers is not in the document]
Comment (qeamlgit) [reply src/main/java/com/tallcraft/chatreplay/ChatReplay.java:177]: Yes, I'll update the code soon.

**Review by Trikolon — APPROVED** [review]
Thanks!

**Pull request merged**